packages/react/src/components/Rating/Rating.base.tsx

```diff
@@ -165,8 +168,9 @@ export const RatingBase: React.FunctionComponent<IRatingProps> = React.forwardRe
       <div
         ref={ref}
         className={css('ms-Rating-star', classNames.root, rootSizeClass)}
-        aria-label={!readOnly ? ariaLabel : ''}
+        aria-label={!readOnly ? ariaLabel : undefined}
         id={id}
+        role={!readOnly ? 'group' : undefined}
```

Comment:
Should this be `radiogroup`? `group` is very generic.

Reply:
Ah, yup! Totally missed that 😄